app/__init__.py

```diff
@@ -1,10 +1,23 @@
 from flask import Flask
 from .routes.planet_routes import planets_bp
+from .db import db, migrate
+from .models import planet
+import os
 
-
-def create_app(test_config=None):
+def create_app(config=None):
+    # __name__ stores the name of the module we're in
     app = Flask(__name__)
-    # registered the planets blueprint with the app
+
+    app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+    app.config['SQLALCHEMY_DATABASE_URI'] = os.environ.get('SQLALCHEMY_DATABASE_URI')
```

Comment:
👍 Loads the connection string from the environment (provided with our `.env` during development).